docs/robot-operation/installation_drivers.md

```diff
@@ -65,12 +65,8 @@ cd spinnaker-2.4.0.143-amd64/
 sudo dpkg -i libgentl_2.4.0.143_amd64.deb libspinnaker_2.4.0.143_amd64.deb libspinnaker-dev_2.4.0.143_amd64.deb libspinnaker-c_2.4.0.143_amd64.deb libspinnaker-c-dev_2.4.0.143_amd64.deb
 
 # Setting up FLIR driver (libspinnaker). Enter username when asked!
-sudo ./configure_gentl_paths.sh 64
-sudo ./configure_usbfs.sh
-sudo ./configure_spinnaker.sh
-sudo ./configure_spinnaker_paths.sh
+sudo ./install_spinnaker.sh
```

Comment:
The point of not using the install_spinnaker.sh is that this script installs more than actually needed (e.g. libspinvideo*)

Reply:
Yes, I see the point here and agree with you.